payjoin-relay/src/main.rs

```diff
@@ -218,6 +221,15 @@ impl HandlerError {
                 error!("Internal server error: {}", e);
                 *res.status_mut() = StatusCode::INTERNAL_SERVER_ERROR
             }
+            HandlerError::OhttpKeyRejection(e) => {
+                const OHTTP_KEY_REJECTION_RES_JSON: &str = r#"{"type":"https://iana.org/assignments/http-problem-types#ohttp-key", "title": "key identifier unknown"}"#;
```

Comment:
i think the title is a bit opaque, maybe its worth mentioning the specific problematic param ie `ohttp_config`

Reply:
I agree, it would be instructive to show something like "Key identifier unknown. Refresh your payjoin directory configuration at $OHTTP_GATEWAY/ohttp-config" or similar.

I think this is a separate task for payjoin-cli, and eventually an implementation library doing I/O.